annotation/tests/test_tasks_services.py

```diff
@@ -794,9 +790,9 @@ def test_filter_tasks_db_no_files_or_jobs(
 )
 def test_create_tasks(
     mock_session: Mock,
-    tasks: List[Dict[str, Union[Set[int], int]]],
+    tasks: Dict[str, Union[int, set]],
     job_id: int,
-    expected_user_ids: Set[int],
+    expected_user_ids: set,
```

Comment:
explicit data type for containers

Reply:
It was okay, before merging. I'll change it again.

Reply:
still there